artichoke-backend/src/extn/core/enumerable/enumerable.rb

```diff
@@ -0,0 +1,450 @@
+##
+# Enumerable
+#
+module Enumerable
```

Comment:
can we alphabetize the methods defined in this source file? I find it easier to see what is defined here vs. what we still need to implement if the method declarations are sorted.